### PR TITLE
Patients features upgrade

### PR DESCRIPTION
**Pull request by BruceZZhao**
1. upgrade journal search to align with treeview
2. upgrade exercise recommendation by providing button to jump to excercises
3. upgrade patient dashboard by adding another bar chart
4. upgrade mhwp dashboard by listing user information above mood score
5. fixed several bugs

### CODE REVIEW

**Review by tt01924 — APPROVED** [review]

File: database/initDBwithDummyData_FeatureTest.py
Comment (timfarkas) [inline database/initDBwithDummyData_FeatureTest.py:1]: was the restoring intentional?

Comment (timfarkas) [reply database/initDBwithDummyData_FeatureTest.py:1]: i'd suggest sticking to the unified dummy data as it's easier to test the whole app in one go that way

Comment (BruceZZhao) [reply database/initDBwithDummyData_FeatureTest.py:1]: Nope, the restoring seems to be automatic. Let me see how to delete it

Comment (BruceZZhao) [reply database/initDBwithDummyData_FeatureTest.py:1]: File deleted

**Review by timfarkas — APPROVED** [review]

**Pull request merged**